orttraining/orttraining/python/training/onnxblock/__init__.py

```diff
@@ -7,5 +7,6 @@
 from . import loss, optim
 from .building_blocks import Block
 from .checkpoint_utils import load_checkpoint_to_model, save_checkpoint
+from .engine import Module, Optimizer
```

Comment:
Why is the execution code defined inside `onnxblock`?

Reply:
so we can call them directly like that : from onnxruntime.training.onnxblock import Module, Optimizer